fastlane/Fastfile

```diff
@@ -142,4 +199,43 @@ platform :ios do
 
   end
 
+  desc 'This function retrives the written version from the staged VERSION file from the repo'
+  def get_version_from_file
+
+    version_file = File.read("VERSION")
+
+    lines = version_file.split("\n")
+    results = lines.map do |line|
```

Comment:
Why don't you directly use `lines[0]` and apply all logic to it since at the end only `results[0]` is used?